### PR TITLE
Modify api endpoint for the 1st chart (Pending applications)

### LINKED ISSUE #75: Update backend query for Pending applications
@tbille commented on [Thu Sep 30 2021](https://github.com/canonical-web-and-design/workplace-engineering-squad/issues/127)

See figma :)

### PR DESCRIPTION
**Pull request by andesol**
## Done
Modified `api/application_review/pending` so it returns data for the pending chart
## QA
- Run server as Peter: `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- View charts on http://0.0.0.0:8501/api/application_review/pending
- Verify it returns `json`
## Issue
Resolves #75

### CODE REVIEW

File: webapp/application_review/constants.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "48_hours",
+    "72_hours",
+    "1_week",
+    "2_weeks",
+    "1_month",
+    "3_months",
+    "more_than_3_months",
+)
+
+age_categories_hours = dict(
```
Comment (jpmartinspt) [inline webapp/application_review/constants.py:12]: I think this one deserves a code comment.

File: webapp/application_review/helpers.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    for key, value in age_categories_hours.items():
+        if hours <= value:
+            return key
+
+
+def build_empty_categories_dict():
+    return dict(zip(age_categories, (0,) * len(age_categories)))
+
+
+def calc_hours(dt1, dt2):
```
Comment (jpmartinspt) [inline webapp/application_review/helpers.py:18]: Can we name this `calculate_hours`?

Comment (jpmartinspt) [reply webapp/application_review/helpers.py:18]: Can we also rename `dt1` and `dt2`  to something more readable?

File: webapp/application_review/helpers.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+
+def get_age_category(hours):
+    for key, value in age_categories_hours.items():
+        if hours <= value:
+            return key
+
+
+def build_empty_categories_dict():
+    return dict(zip(age_categories, (0,) * len(age_categories)))
```
[the remark this answers is not in the document]
Comment (andesol) [reply webapp/application_review/helpers.py]: It does! Changed

**Pull request merged**